### ISSUE TITLE
99-aclidswitch.rules file incorrectly references /usr/bin instead of /usr/local/bin

### ISSUE THREAD

**Issue by kmanwar89** [orschiro/aclidswitch#5]
Hi orschiro,

Just to let you know, the 99-aclidswitch.rules file has /usr/bin as the location of the executable, rather than /usr/local/bin.  You may want to change that, if you're still maintaining this project.

I'm also having an issue in getting it to work at all...all my files are in the correct place with correct executable bits, so I'm not sure what's going on.   Still troubleshooting, though.

**Comment by orschiro** [comment]
Dear @kmanwar89,

Thank you for your report!

I updated the files in https://github.com/orschiro/aclidswitch/commit/324564d187946ac9a45548053352a13cca25e339. 

Warmly,

-Robert

**Comment by kmanwar89** [comment]
I see an error message when I manually run the command from the terminal (see screenshot) but it still performs the action as expected.  It does not automatically perform it's action on plug/unplug, though.  

The only action that is /partially/ performs correctly is turning the screen off (or suspending, I can't tell which) when the lid is closed on battery power, but I can't tell which it's doing, or if it's doing it properly.

![20160822-09 17 58_1920x1080_scrot](https://cloud.githubusercontent.com/assets/11323060/17856185/3aafcbc8-684a-11e6-93d6-ad471a874412.png)

**Comment by orschiro** [comment]
Do you still see the permissions problem when running `sudo aclidswitch AC`, for instance?

**Comment by kmanwar89** [comment]
No I don't, but it still doesn't activate automatically with power plug or unplug.

**Comment by orschiro** [comment]
Right. So, it does work manually then. 

Now, we have to find out why it is not triggered automatically be the Udev rule.

You placed the rules into the correct folder, correct?

![image](https://cloud.githubusercontent.com/assets/170947/17860162/bc303ec0-688c-11e6-9498-36a7f2c250dc.png)

Can you test another rule [1] and see if you maybe have a general problem with Udev?

Thanks!

-Robert

[1] https://wiki.archlinux.org/index.php/Udev#Writing_udev_rules

**Comment by kmanwar89** [comment]
> Can you test another rule [1] and see if you maybe have a general problem with Udev?

I've actually got other udev rules already in place, and they work without issue.  I'm actually testing the `light-git` package to handle the AC/battery brightness functionality and it works without issue.

> You placed the rules into the correct folder, correct?

Yup! Followed your instructions to the letter (see screenshot). I agree that it's an issue somewhere with `udev`, but it's confusing since other rules work fine.  I already edited the udev rule to include the change to the file path, but that didn't seem to fix it.

![20160822-11 40 19_1920x1080_scrot](https://cloud.githubusercontent.com/assets/11323060/17861023/6fa39b3a-685d-11e6-92c3-d8bfda375321.png)
